packages/system-data/src/collections/index.ts

```diff
@@ -15,5 +15,7 @@ export const systemCollectionNames: string[] = (systemData['data'] as DataCollec
 );
 
 export function isSystemCollection(collection: string): boolean {
+	if (!collection.startsWith('directus_')) return false;
```

Comment:
Not sure about this check, feels like there is something underlying going wrong that we should look to fix instead.

Reply:
Nothing was wrong with the original function, this addition was done to speed up lookup time for non directus collections.